core/modules/dashboard/includes/block.welcome.inc

```diff
@@ -131,6 +158,9 @@ class DashboardWelcomeBlock extends Block {
     $html_links = array();
     foreach ($links as $path => $label) {
       if (url_is_external($path) || $path === '<front>') {
+        if ($path !== '<front>') {
+          $label .= '<span class="external"></span>';
+        }
```

Comment:
I don't think we should be trying to tackle the problem of external links only on the welcome block. We have dozens of links to BackdropCMS.org in the code elsewhere that we don't have an external link icon for.

Reply:
ok. moved css to dashboard module.